book/src/02_2_software.md

```diff
@@ -13,8 +13,8 @@ Furthermore, for ESP32-C3, a *nightly* version of the Rust toolchain is currentl
 ✅ Install nightly Rust and add support for the target architecture using the following console commands:
 
 ```console
-$ rustup install nightly
-$ rustup component add rust-src --toolchain nightly
+$ rustup install nightly-2022-03-10
+$ rustup component add rust-src --toolchain nightly-2022-03-10
 ```
 
```

Comment:
~~I wonder if we should add here that the exercise come with a toolchain file that will do that work for them?~~

Ignore that, it was in optional.